analyzer/queries/queries.go

```diff
@@ -808,4 +834,104 @@ var (
       source_files = $5
     WHERE
       runtime = $1 AND contract_address = $2`
+
+	RuntimeEvmVerifiedContractTxs = `
+    WITH abi_contracts AS (
+      SELECT 
+        runtime,
+        contract_address AS addr,
+        abi
+      FROM chain.evm_contracts
+      WHERE
+        runtime = $1 AND verification_info_downloaded_at IS NOT NULL
+    )
+    SELECT 
+      abi_contracts.addr,
+      abi_contracts.abi,
+      txs.tx_hash,
+      txs.body->'data',
+      txs.error_message_raw,
+    FROM abi_contracts 
+    JOIN chain.runtime_transactions as txs ON
+      txs.runtime = abi_contracts.runtime AND
+      txs.to = abi_contracts.addr AND
+      txs.method = 'evm.Call' -- note: does not include evm.Create txs
+    WHERE
+      runtime = $1 AND
```

Comment:
Should/can we further narrow the selection here by checking for empty `body` and `error_message_raw`?

Reply:
good idea, added!